server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -326,8 +325,8 @@ private void addProductContentToMap(Map<String, Content> contentMap, Product pro
 
             if (existing != null && !content.equals(existing)) {
                 log.warn(
-                    "Multiple versions of the same content found on a single subscription: {}.{}",
-                    (content.getOwner() != null ? content.getOwner().getId() : null), content.getId()
+                    "Multiple versions of the same content found on a single subscription: {}",
+                    content.getId()
```

Comment:
We log a warning that multiple versions of the same content were found, but there's no info on what that means to the overall refresh process. Does one content replace the other? Are both versions still going to be available? If there is not going to be an impact to the resulting subscriptions, then perhaps we should change the log level to debug?